.organizationmap

```diff
@@ -913,7 +1061,9 @@ Unaffiliated <no@organization.net> Dmitriy Rabotjagov <noonedeadpunk@ya.ru>
 Unaffiliated <no@organization.net> Dmitry Plyakin <dplyakin@gmail.com>
 Unaffiliated <no@organization.net> Dominik Hannen <cantares1+github@gmail.com>
 Unaffiliated <no@organization.net> Dongdong Tao <tdd21151186@gmail.com>
+Unaffiliated <no@organization.net> Dongsheng Yang <dongsheng.yang.linux@gmail.com>
```

Comment:
I think this should linked to EasyStack. cc @yangdongsheng

Reply:
Yea, thanx @idryomov  ：）